src/commands/redis_cmd.cc

```diff
@@ -103,24 +104,28 @@ Status ParseTTL(const std::vector<std::string> &args, std::unordered_map<std::st
   int ttl = 0;
   int64_t expire = 0;
   bool last_arg = false;
+  int ex_exist = 0, exat_exist = 0, pxat_exist = 0, px_exist = 0;
```

Comment:
```suggestion
  bool has_ex = false, has_exat = false, has_pxat = false, has_px = false;
```

You can still use `has_ex + ... + > 1` below. I think it's an idiom in C++. cc @PragmaTwice